_episodes/05-loop.md

```diff
@@ -199,7 +199,7 @@ done
 because then the first time through the loop,
 when `$filename` expanded to `basilisk.dat`, the shell would try to run `basilisk.dat` as a program.
 Finally,
-the `head` and `tail` combination selects lines 81-100 from whatever file is being processed.
+the `head` and `tail` combination selects a set of lines from whatever file is being processed.
```

Comment:
I see the problem. But just saying "a set of lines" is not very helpful (even though technically more correct). Maybe something like:

> selects lines 81-100 from whatever file is being processed (assuming the file has at least 100 lines)